test/integration/components/datepicker.js

```diff
@@ -14,7 +14,11 @@ context('Datepicker', function() {
     },
     template: hbs`
       <button class="button--blue u-margin--t-16 u-margin--l-16">
-        {{formatDateTime date "LONG" defaultHtml="Select Date"}}
+        {{#unless selectedMonth}}
```

Comment:
for this test let's do `{{#if selectedMonth}}` and `{{#if date}}`  only one should be non-null at a time, so let's test that.